LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2596,6 +2596,78 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chipset Fan", 3));
                         break;
 
+                    case Model.X570_Phantom_Gaming_4: // NCT6796D (-R?)
+                        // internal on NCT6796D have a 1/2 voltage divider (by way of two 34kOhm resistors)
+                        // "Six internal signals connected to the power supplies (CPUVCORE, AVSB, VBAT, VTT, 3VSB, 3VCC)"
+                        // "All the internal inputs of the ADC, AVSB, VBAT, 3VSB, 3VCC utilize an integrated voltage divider
+                        //  with both resistors equal to 34kOhm"
+                        // it seems that VTT doesn't actually have the 1/2 divider
+
+                        // external sources can have whatever divider that gets them in the 0V to 2.048V range
+
+                        // assuming Vf = 0, then Ri = R1 and Rf = R2 (from voltage divider equation)
+
+                        v.Add(new Voltage("Vcore", 0, 1, 1));
+                        v.Add(new Voltage("+5V", 1, 2, 1));
+                        v.Add(new Voltage("AVCC", 2, 1, 1));
+                        v.Add(new Voltage("+3.3V", 3, 1, 1));
+                        v.Add(new Voltage("+12V", 4, 56, 10));
+                        v.Add(new Voltage("VDDCR_SOC", 5));
+                        v.Add(new Voltage("DRAM", 6));
+                        v.Add(new Voltage("3VSB", 7, 1, 1));
+                        v.Add(new Voltage("VBat", 8, 1, 1));
+                        v.Add(new Voltage("VTT", 9));
+                        v.Add(new Voltage("Voltage #11", 10, true)); // unknown. VIN5 pin
+                        v.Add(new Voltage("VPPM", 11, 3, 1));
+                        v.Add(new Voltage("PREM_VDDCR_SOC", 12));
+                        v.Add(new Voltage("VDDP", 13));
+                        v.Add(new Voltage("CPU +1.8V", 14, 1, 1));
+                        v.Add(new Voltage("Voltage #16", 15, true)); // unknown. VIN9 pin
+
+                        t.Add(new Temperature("CPU", 9)); // AKA SMBUSMASTER0
+                        t.Add(new Temperature("SB (Chipset)", 10)); // AKA SMBUSMASTER1
```

Comment:
Figured I'd chip in to keep this PR moving a bit. These name changes might be suggested as this seems to be the preferred names:

`3VSB` -> `+3V Standby`
`VBat` -> `CMOS Battery`
`CPU +1.8V` -> `CPU 1.08V`
`SB (Chipset)` -> `Chipset`
`SB Fan` -> `Chipset Fan`

Again, not an official suggestion, but if the contributions I've done with my motherboard additions are anything to go by (#1364, #1360, and #1343 are some notable ones), I expect this will likely be requested.

Reply:
Yes, that would be fine! I also think we need to come up with a more descriptive name for these:
VDDCR_SOC
VPPM
PREM_VDDCR_SOC